### DISCUSSION TITLE
Image planes not showing!

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#3142] [General]
Hi Everyone,

I've noticed when testing my model on a Pixel 6 Pro the image plane disappears, Is this a common issue? 

If anyone could please help is would be greatly appreciated  

![image](https://user-images.githubusercontent.com/90825510/150533294-f05b0b7d-173d-466f-9bba-95364e0537ce.png)

![image](https://user-images.githubusercontent.com/90825510/150533377-7fd11734-1307-47b6-8bef-6e6d17866fd6.png)

**Comment by samaneh-kazemi** [comment]
> Hi Everyone,
> 
> 
> 
> I've noticed when testing my model on a Pixel 6 Pro the image plane disappears, Is this a common issue? 
> 
> 
> 
> If anyone could please help is would be greatly appreciated  
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/90825510/150533294-f05b0b7d-173d-466f-9bba-95364e0537ce.png)
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/90825510/150533377-7fd11734-1307-47b6-8bef-6e6d17866fd6.png)
> 
> 
> 
> 

Hi,
Thanks for reaching out. Can you please share a repro link? Which version of model-viewer do you use?

**Reply by elalish** [reply]
@Jasonp8000 What do you mean by the "image plane"? Can you circle the difference you're referring to in your screenshots? Also, for the Pixel 6, which AR mode are you using? Try restricting `ar-modes` to just `webxr` or `scene-viewer` to check if you get different results if one isn't working at all.

**Reply by Jasonp8000** [reply]
Thanks for the reply, I have tried isolating different ar modes with no success unfortunately.

**Reply by Jasonp8000** [reply]
This is the image I'm referring too 

![image](https://user-images.githubusercontent.com/90825510/150626419-430ec5cf-94de-4679-b72c-f4b38ca502b9.jpeg)

**Reply by elalish** [reply]
well,  what I'm looking for are the details of your "no success". *some* behavior should change if you select only a single AR mode and go through each different one.  I need to know what's  happening for you in  order  to diagnose this.
